components/status/StatusActionsMore.vue

```diff
@@ -31,19 +31,23 @@ const { t } = useI18n()
 const userSettings = useUserSettings()
 const useStarFavoriteIcon = usePreferences('useStarFavoriteIcon')
 
+const isSingleInstanceServer = useRuntimeConfig().public.singleInstance
```

Comment:
I think it makes sense to pull this out into `route.ts` (see below) since it's only used in the call to `getStatusRoute`

Reply:
I was wondering about that, but that file seemed to be utility only, which is why I passed it in.